checkpointctl.go

```diff
@@ -77,12 +76,6 @@ func setupInspect() *cobra.Command {
 	}
 	flags := cmd.Flags()
 
-	flags.BoolVar(
-		&stats,
-		"print-stats",
-		false,
-		"Display checkpoint statistics",
-	)
```

Comment:
This change is not directly related to the feature displaying file descriptors. Would it be possible to create a new pull request?

Reply:
Righto, I'll move both the fix commits to a separate PR